modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -222,9 +222,9 @@ When this section is empty, {prod-short} only loads the implicit components.
 |N/A
 
 3+>s|Total
->s|8.5 GiB
->s|1.1 GiB
+>s|9 GiB
```

Comment:
```suggestion
>s|8.9 GiB
```

Reply:
it is 9048 according to my calculations, no?

Reply:
Memory limit: 256+1000+256+4000+256+512+512+256+256+1000+300+300=8904=8.9 GiB
Memory request: 64+512+32+128+64+64+64+32+32+100+20+50+20=1182=1.2 GiB
CPU limit: 500+1000+500+1000+500+500+500+500+500+1000+200+200=6900=6.9
CPU request: 100+100+100+100+50+100+100+100+100+250+100+10+100=1310=1.3